### DISCUSSION TITLE
New text input fields

### DISCUSSION THREAD

**Discussion by NemirStephan** [Legal-and-General/canopy#322] [Ideas]
I've been reviewing the designs (in progress) for the new change address component and it looks like we will need a new textInput component

https://legalandgeneral.invisionapp.com/console/Account-Details-cknlj6bn71jia01ym13rjlqh8/cknljacp61mt601yim73x00uw/play#project_console

<img width="426" alt="Screenshot 2021-04-19 at 17 36 42" src="https://user-images.githubusercontent.com/22469863/115272012-d55e5580-a135-11eb-8521-3bce4a2a4013.png">

What are peoples thoughts?

**Comment by owensgit** [comment]
Do you mean for a text area? If so, yes I think this is a good idea 👍

**Reply by NemirStephan** [reply]
I did, but I also realised the designs changed so we don't yet have a need for it although i'm sure we will at some stage
